clevertap-core/src/main/java/com/clevertap/android/sdk/db/DBManager.kt

```diff
@@ -18,6 +18,8 @@ internal class DBManager(
 ) : BaseDatabaseManager {
 
     private var dbAdapter: DBAdapter? = null
+    private val userEventLogsThreshold = 11_520 //12.59 MB table and index size
```

Comment:
This is a bit confusing since the threshold is a number of rows but the comment says MB size.
I believe it would be better to do something along the lines of:
 ```
private val numberOfEventLogsPerUser = 2_304 // (2048 events + 256 profile props = 1 user)
private val userEventLogsThreshold = 11_520 // (5 * 2_304)
private val numberOfRowsToCleanupForUserEventLogs = userEventLogsThreshold
```

Reply:
resolved by @vasct